### PR TITLE
Add provider_id and provider_type to system_profile

### PR DESCRIPTION
**Pull request by xiangce**
- See ESSNTL-3114 for context, I'd like to add the following new keys to the `system_profile`
- provider_id and provider_type

Signed-off-by: Xiangce Liu <xiangceliu@redhat.com>

### CODE REVIEW

File: src/puptoo/process/profile.py [hunk 1]
```diff
@@ -5,7 +5,9 @@
 
 from insights import make_metadata, rule, run
 from insights.combiners.cloud_provider import CloudProvider
+from insights.combiners.cloud_instance import CloudInstance
 from insights.combiners.redhat_release import RedHatRelease
+from insights.combiners.os_release import OSRelease
```
Comment (r14chandra) [inline src/puptoo/process/profile.py:10]: @xiangce Is import `OSRelease` required here? As it is not used anywhere.

Comment (xiangce) [reply src/puptoo/process/profile.py]: Sorry for this, I just removed it in the latest commit, please have a look.

**Review by r14chandra — APPROVED** [review]
Thanks @xiangce for the PR. LGTM!

**Comment by xiangce** [comment]
@r14chandra , Thanks for reviewing!

**Pull request merged**